### PR TITLE
Hide the login box by default and only show it for private repo

### PR DESCRIPTION
**Pull request by barnettwilliam**
Hide the login box by default and only show it when a private repo is given. This prevents the login box's brief display when loading a private repo mdenet/educationplatform-docker#2.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
 <body class="h-100" onresize="fit()" onload="fit();updateGutterVisibility();">
     <div id="preloader"><img src="images/preloader.gif" width="100px"></div>
 
-    <div id="login" style="display:inline">
+    <div id="login" style="display:none">
```
Comment (szschaler) [inline public/index.html:27]: Why is this file needed at all? Shouldn't it just use the file imported from the education platform repository?

**Review by szschaler — CHANGES_REQUESTED** [review]
Can you clarify why this `index.html` file is needed at all?

File: public/index.html (hunk 1 above)
Comment (barnettwilliam) [reply public/index.html:27]: For the dockerised playground there there is some redirect functionality to the list of example activities if no activity file is specified that is added to the html. This could be done differently to avoid the duplication of the playground index.html.

**Review by szschaler — APPROVED** [review]
